packages/dcos-integration-test/extra/test_ec2.py

```diff
@@ -145,7 +82,17 @@ def test_move_external_volume_to_new_agent(cluster):
         cluster.deploy_marathon_app(read_app, **deploy_kwargs)
         cluster.destroy_marathon_app(read_app['id'])
     finally:
-        try:
-            _delete_ec2_volume(test_label)
-        except Exception as ex:
-            raise Exception("Failed to clean up volume {}: {}".format(test_label, ex)) from ex
```

Comment:
It's raising an exception with a clear explanation that might be understandable and actionable to someone who's not familiar with the test, which means potentially fewer questions for me about what happened, since in my experience people usually don't spend too much effort debugging a test if they don't own it. It's raised `from ex`, so it'll include the traceback of the exception that caused it. No detail is lost here.